### DISCUSSION TITLE
Finding the location of the NodalMaxValue

### DISCUSSION THREAD

**Discussion by robfairh** [idaholab/moose#18737] [Q&A General] [answered]
Hi All,

I hope this is a quick question.
I was trying to find a postprocessor for finding the max value of a variable in a certain domain and its coordinates.
Is there an easy way of doing this?
I am looking for something similar to ```TimeExtremeValue``` that can report either the extreme value or the time at which it occurred. What I need is the location instead of the time.

Thank you,
Roberto

**Comment by aeslaughter** [comment, marked as answer]
I do not think that this exists, but it could be created within a single Reporter object: https://mooseframework.inl.gov/moose/syntax/Reporters/.

**Reply by robfairh** [reply]
Great! Thank you.
